Scroller/EditableWrapper.js

```diff
@@ -165,8 +168,15 @@ const EditableWrapper = (props) => {
 
 				// Set the selected item's offset to css variable
 				const offset = (toIndex - fromIndex) * itemWidth;
-				wrapperRef.current.style.setProperty('--selected-item-offset', offset + 'px');
-
+				if (trigger === 'scroll') {
+					if (toIndex === dataSize - 1) scrolloffset = offset;
+					if (toIndex === 0) scrolloffset = (-1) * itemWidth * rearrangedItems.length;
+					wrapperRef.current.style.transition = 'transform 0';
```

Comment:
This code works?? It looks same without this.
Is the wrapperRef correct, not the selectedItem ?

Reply:
It was my mistake. It's removed.